dissect/target/loaders/log.py

```diff
@@ -1,13 +1,23 @@
+from __future__ import annotations
+
 import urllib
 from pathlib import Path
 from typing import Union
 
 from dissect.target import Target
-from dissect.target.filesystem import VirtualFilesystem
+from dissect.target.filesystem import RootFilesystem, VirtualFilesystem
 from dissect.target.loader import Loader
 
+logloader_option_hint = "hint"
+
 
 class LogLoader(Loader):
+    """Load separate log files without a target. Usage: target-query /evtx/* -L log -f evtx (-L log -h for options)"""
+
+    __loader_args__ = {
+        logloader_option_hint: {"help": "Hint for file type"},
```

Comment:
It's not great to have to define these as dictionaries. Can't you re-use some of the code behind `@arg`?